Link community resources (#436)
subspace/subspace-docs: docs/participate/community_resources/community-build-tools.md

---

> title: Community Build Tools

Reviewer: ```suggestion
title: Community Built Tools
```

sidebar_position: 2

> description: essential info to file systems. partitioning, and mounting.

Reviewer: Should this be updated? I'm not sure where it's used?

Author: This is used for indexing, it's better to update it. Thanks for catching it!

keywords:
    - Subspace Network

>     - Community build tools

Reviewer: ```suggestion
    - Community built tools
```

---

> ### Community-maintained list of userful, third-party tools

Reviewer: ```suggestion
### Community-maintained list of useful, third-party tools
```

> [Wolfrage76's](https://github.com/wolfrage76) **list of third-party tools**. 

Reviewer: ```suggestion
[Wolfrage76](https://github.com/wolfrage76)'s **list of third-party tools**. 
```

> - Link to the [Website.](https://subspace.ifhya.com/)

Reviewer: ```suggestion
- Link to the [website](https://subspace.ifhya.com/).
```

- Contact **Wolfrage** in the Subspace Discord to add your tool.

<iframe src="https://subspace.ifhya.com/" width="100%" height="650px" frameborder="0" scrolling="auto"></iframe>

### Plotting speed measurement tool

> [Hakehardware's](https://github.com/hakehardware) **Plot Report** that allows you to measure your plotting speed and adds it to the table for comparison with other hardware plotting speeds. 

Reviewer: ```suggestion
[Hakehardware](https://github.com/hakehardware)'s **Plot Report** that allows you to measure your plotting speed and adds it to a table for comparison with other hardware plotting speeds. 
```

> - Link to [GitHub](https://github.com/hakehardware/plotreport)

Reviewer: ```suggestion
- Link to [GitHub](https://github.com/hakehardware/plotreport).
```

> - Link to the [Website](https://plotreport.hakedev.com/)

Reviewer: ```suggestion
- Link to the [website](https://plotreport.hakedev.com/).
```

> - Link to the [Youtube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg)

Reviewer: ```suggestion
- Link to the [YouTube channel](https://www.youtube.com/channel/UCakvG7QQp4oL0Rtpiei1yKg).
```
